tools/pythonpkg/src/vector_conversion.cpp

```diff
@@ -396,8 +396,16 @@ void VectorConversion::BindPandas(py::handle original_df, vector<PandasColumnBin
 					bind_data.pandas_type = PandasType::CATEGORY;
 					auto enum_name = string(py::str(df_columns[col_idx]));
 					vector<string> enum_entries = py::cast<vector<string>>(categories);
+					idx_t size = enum_entries.size();
+					Vector enum_entries_vec(LogicalType::VARCHAR);
```

Comment:
No need to allocate and then call resize, we can immediately allocate with the correct size:

`Vector enum_entries_vec(LogicalType::VARCHAR, size);`